actionview/lib/action_view/rendering.rb

```diff
@@ -92,12 +92,16 @@ def rendered_format
       # Find and render a template based on the options given.
       # :api: private
       def _render_template(options) #:nodoc:
-        variant = options[:variant]
+        variant = options.delete(:variant)
+        assigns = options.delete(:assigns)
+        context = view_context
+
+        context.assign assigns if assigns
```

Comment:
Also I'd like to kill the `view_assigns` dependency on Action View entirely and always receive a `assigns` option, but not sure if it is possible. Worth to check